deploy/local.yaml.example

```diff
@@ -117,6 +117,8 @@ data:
     #          container: "containerName"
     #          pod: "podName"
     #          deployment: "deploymentName"
+    # integration definition files required to map metrics to entities
```

Comment:
Shouldn't in k8s this be commented? The default behaviour should be `standalone` so `DefinitionFilesPath` is not used, no?

Reply:
if we are packaging POMI with the agent it will always be standalone: false and we will need a path to the definition files, no? Even in K8s this is how it's going to work I think, right?

Reply:
Yes, once it is packaged with the agent that is going to be the configuration, but currently since it is not if `standalone: false` the pod created by the deployment will start and exit each time entering in a `crashloopbackoff` state

Reply:
ok I see your point. I'll change it back for now